### PR TITLE
Move CacheStatsMBean to Airlift

### PR DESCRIPTION
**Pull request by alexjo2144**
With the airlift dependency bumped we can remove the copy in presto: https://github.com/prestosql/presto/commit/36a6abc4b63157ae399e543e45eeb98d80e7604f

### CODE REVIEW

**Comment by findepi** [comment]
> We might want to move this to Airlift since it's not specifically related to plugins.

then i guess this would go into https://github.com/airlift/airlift/tree/4b1c83d0bd47d21649b200829f2ebdbde5544792/jmx/src/main/java/io/airlift/jmx, just like `StackTraceMBean` there. 

@alexjo2144 would you?

**Comment by electrum** [comment]
@martint agrees we should move it there

**Comment by alexjo2144** [comment]
Can do

**Comment by alexjo2144** [comment]
https://github.com/airlift/airlift/pull/822

**Comment by alexjo2144** [comment]
Rolled back the changes but added a `@Deprecated` annotation as a reminder to remove after pulling in Airlift 194.

File: presto-main/src/main/java/io/prestosql/sql/gen/CacheStatsMBean.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 import static java.util.Objects.requireNonNull;
 
+@Deprecated // TODO: Remove after upgrading to Airlift 194
```
Comment (electrum) [inline presto-main/src/main/java/io/prestosql/sql/gen/CacheStatsMBean.java]: Nit: 0.194

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**